esmvaltool/diag_scripts/droughts/constants.py

```diff
@@ -0,0 +1,75 @@
+"""Constants that might be usefull by any drought diagnostic."""
+
+from iris.coords import AuxCoord
+
+# fmt: off
+DENSITY = AuxCoord(1000, long_name="density", units="kg m-3")
+
+FNAME_FORMAT = "{project}_{reference_dataset}_{mip}_{exp}_{ensemble}_"
+"{short_name}_{start_year}-{end_year}"
+CMIP6_FNAME = "{project}_{dataset}_{mip}_{exp}_{ensemble}_{short_name}_"
+"{grid}_{start_year}-{end_year}"
+OBS_FNAME = "{project}_{dataset}_{type}_{version}_{mip}_{short_name}_"
+"{start_year}-{end_year}"
+
+CONTINENTAL_REGIONS = {
+    "Global": ["GLO"],  # global
+    "North America": ["GIC", "NWN", "NEN", "WNA", "CNA", "ENA"],
+    "Central America": ["NCA", "SCA", "CAR"],
+    "Southern America": ["NWS", "NSA", "NES", "SAM", "SWS", "SES", "SSA"],
+    "Europe": ["NEU", "WCE", "EEU", "MED"],
+    "Africa": ["SAH", "WAF", "CAF", "NEAF", "SEAF", "WSAF", "ESAF", "MDG"],
+    "Asia": ["RAR", "WSB", "ESB", "RFE", "WCA", "ECA", "TIB", "EAS", "ARP",
+             "SAS", "SEA"],
+    "Australia": ["NAU", "CAU", "EAU", "SAU", "NZ", "WAN", "EAN"],
+}
+
+HEX_POSITIONS = {
+    "NWN": [2, 0], "NEN": [4, 0], "GIC": [6.5, -0.5], "NEU": [14, 0],
+    "RAR": [20, 0], "WNA": [1, 1], "CNA": [3, 1], "ENA": [5, 1],
+    "WCE": [13, 1], "EEU": [15, 1], "WSB": [17, 1], "ESB": [19, 1],
+    "RFE": [21, 1], "NCA": [2, 2], "MED": [14, 2], "WCA": [16, 2],
+    "ECA": [18, 2], "TIB": [20, 2], "EAS": [22, 2], "SCA": [3, 3],
+    # "CAR": [5, 3],
```

Comment:
```suggestion
    # "CAR": [5, 3],
```
why is this commented out?